DirectEmployers/MyJobs
# [PD-3020] Unquote already url-encoded campaign tags to avoid re-quoting them when building a job url.

## What's this PR do?
Un-quotes urlencoded query parameters stored in cookies to avoid duplicate urlencoding on the parameter.

## Where should the reviewer start?
In `seo_extras.py`, unquoting is now used on values coming from the the already url-encoded cookies.

## How should this be manually tested?
To reproduce the issue:
1. Go to http://www.my.jobs?utm_campaign='random'
2. Navigate to an apply page for a job, confirm that the apply link is on the job and properly urlencoded.
3. Click the search button on that page.
4. Select a job from the search results.
5. Confirm that the `utm_campaign` parameter on the apply link is now double encoded.

Checking out the branch and repeating these steps should result in the apply link in step 5 no longer being urlencoded.
